src/System.Net.Primitives/src/System/Net/SocketException.netstandard17.cs

```diff
@@ -15,8 +15,10 @@ protected SocketException(SerializationInfo serializationInfo, StreamingContext
         {
             if (GlobalLog.IsEnabled)
             {
-                GlobalLog.Print("SocketException::.ctor(serialized) " + NativeErrorCode.ToString() + ":" + Message);
+                GlobalLog.Print($"SocketException::.ctor(serialized, SocketErrorCode={SocketErrorCode}):{Message}");
             }
         }
+
+        public override int ErrorCode => base.NativeErrorCode;
```

Comment:
You'll need to add this to the ref as well:
https://github.com/dotnet/corefx/blob/master/src/System.Net.Primitives/ref/System.Net.Primitives.cs#L416
Even though it's an override, we still specify it in the ref, as it can affect certain specific cases, e.g. a type deriving from this type and calling down to the base method.